resources/views/macros/base.twig

```diff
@@ -20,7 +20,12 @@
     <a href="{{ opt.url|default(url('users', {'action': 'view', 'user_id': user.id})) }}"
             {%- if not user.state.arrived %} class="text-muted"{% endif -%}
     >
-        {{ _self.angel() }} {{ user.name }}
+        {{ _self.angel() }} 
+        {% set username = user.name %}
+        {% if(config('enable_show_user_name')) %}
+            {% set username = user.personalData.first_name ~ ' ' ~  user.personalData.last_name %}
+        {% endif %}
+        {{ username }}
```

Comment:
Imho that decision should be extracted to its own macro which also can be used in the navbar template